build.gradle.kts

```diff
@@ -235,5 +235,5 @@ fun releaseNotes(): Provider<String> {
 @Suppress("UNCHECKED_CAST")
 fun getSupportedVersions(): Map<String, Array<String>> {
     return (JsonSlurper()
-        .parse(file("src/main/resources/versions.json")) as Map<String, Map<String, Array<String>>>).getValue("supportedVersions")
+        .parse(providers.fileContents(layout.projectDirectory.file("src/main/resources/versions.json")).asBytes.get()) as Map<String, Map<String, Array<String>>>).getValue("supportedVersions")
```

Comment:
"Applying suggestions on deleted lines is not supported."

Maybe this is more readable?

```kotlin
@Suppress("UNCHECKED_CAST")
fun getSupportedVersions(): Map<String, Array<String>> {
    val versions = providers.fileContents(layout.projectDirectory.file("src/main/resources/versions.json"))
    return (JsonSlurper().parse(versions.asBytes.get()) as Map<String, Map<String, Array<String>>>).getValue("supportedVersions")
}
```

Reply:
I tried to figure out a way to remove the need to hardcode `src/main/resources` You can, but it's quite verbose. You can call `sourceSets.main.map { it.resources .... }`.

Reply:
I like the first suggestion and I applied it.
I also had issues with github suggestions in PR comments today. It wasn't working.